examples/forecasts/basic_example_forecasting.py

```diff
@@ -0,0 +1,56 @@
+from forecasting.models.gluonts_model.mqcnn.mqcnn_model import MQCNNModel
+from forecasting.task.forecasting.dataset import TimeSeriesDataset
+
+import core as ag
+from core.scheduler import FIFOScheduler
```

Comment:
Since this file is intended to show how to use AutoGluon, all imports should import autogluon, eg: `import autogluon.core` instead of `import core`